cyberbrain/frame.py

```diff
@@ -57,6 +57,7 @@ def __init__(
         frame_name: str,
         instructions: dict[int, Instruction],
         offset_to_lineno: dict[int, int],
+        fn_lineno: int,
```

Comment:
I would argue that `def_lineno` is probably a better name for it. Imagine in the future we need to add more types of line numbers (like where it's called, where the function ends). Making the name more explicit is future proof.

Reply:
I'm just worried `def_lineno` might represent the line that the keyword `def` is on, and that isn't always true, cause decorators might exist

Reply:
how about `defined_lineno`?

Reply:
Sgtm